glide.yaml

```diff
@@ -9,3 +9,7 @@ import:
 - package: github.com/spf13/cobra
 - package: github.com/xeipuuv/gojsonschema
   version: d5336c75940ef31c9ceeb0ae64cf92944bccb4ee
+- package: github.com/russross/blackfriday
```

Comment:
should you have called `glide get --test github.com/russross/blackfriday` instead to have those tests deps in `testImports`? https://glide.readthedocs.io/en/latest/glide.yaml/